bot/admin/web/src/app/core/bot-core.module.ts

```diff
@@ -32,8 +32,8 @@ export class BotCoreConfig implements CoreConfig {
   roleMap: Map<UserRole, string[]> = new Map([
     [UserRole.nlpUser, ['/nlp']],
     [UserRole.faqNlpUser, ['/faq/training']],
-    [UserRole.faqBotUser, ['/faq/manangement', '/build/i18n']],
-    [UserRole.botUser, ['/build']],
+    [UserRole.faqBotUser, ['/faq/manangement', '/build/i18n', '/configuration']],
+    [UserRole.botUser, ['/build', '/configuration']],
```

Comment:
Good 👍️ but nlpUser and faqNlpUser need also access to the configuration route so that thet can train all their applications in all namespaces

Reply:
Done